src/win_bindings.rs

```diff
@@ -0,0 +1,452 @@
+// Bindings generated by `windows-bindgen` 0.58.0
```

Comment:
I have an allergy to checking in generated code.  build.rs?

Reply:
Why? `build.rs` means all your downstreams have to build an extra compilation unit and extra dependencies just to regenerate some code that ~never changes, and your CI guards against changes in the generated code.

Reply:
Well, at a minimum, this needs some predefined process for updating.  That might be past of a GitHub action, but checking in generated code without any means to ensure that it continues to track something risks picking up and carrying vulnerabilities.

Reply:
Maybe a scheduled Actions run?

Reply:
That's what I was thinking.  Scripting the update is good, running that script periodically is also good, particularly when you can activate all the usual dependency checkers (dependabot etc...).

Reply:
Added a separate commit with this.

Reply:
So the intention here is that whenever CI fails, we go and recreate an updated version of the bindings and then commit them?

Reply:
Yes, that's the idea. From my experience with other projects using this approach (in particular, chrono), this almost never happens.